dev/devicelab/bin/tasks/build_aar_module_test.dart

```diff
@@ -0,0 +1,225 @@
+// Copyright (c) 2018 The Chromium Authors. All rights reserved.
```

Comment:
```suggestion
// Copyright (c) 2019 The Chromium Authors. All rights reserved.
```

Reply:
nit: I think we've been using `The Flutter Authors` as well.